fancy-team-server/Services/Event/index.ts

```
@@ -1,59 +1,105 @@
-import { Client } from 'pg';
+import {Client} from 'pg';
 import {Event} from "../../entity/Event";
 import {getConnection} from "typeorm";
-import {User} from  '../User';
-import {UserEntity} from "../../entity/User";
+import {createQueryBuilder} from 'typeorm';
 import {UserEventStatus} from "../../entity/UserEventStatus";
+import {EmailService} from "../../services/Email";
+import {UserService} from "../../services/User";
+
+import {UserEventStatusService} from "../../services/UserEventStatus"
+
 
 const client = new Client();
 client.connect();
 
+export class EventService {
 
+    public static findEvent = async (id: number) => {
+        try {
+            const eventRepository = getConnection().getRepository(Event);
+            return await eventRepository.findOne({id: id});
+        }
+        catch (error) {
+            console.log(error);
```

you need to throw the error and not loose it